packages/katello/katello/backup.rb

```diff
@@ -122,7 +122,9 @@ def configure_configs
       feature_configs += ["/var/lib/dhcpd", "/etc/dhcp"] if feature_included?("dhcp")
       feature_configs.push("/usr/share/xml/scap") if feature_included?("openscap")
 
-      backup_configs + feature_configs
+      all_configs = backup_configs + feature_configs
+      all_configs = all_configs.compact
+      all_configs.collect { |path| Dir.glob(path) }.flatten
```

Comment:
Would it make sense to run a uniq on this?

Reply:
I'd not expect duplicates here, but it also won't hurt.